apiserver/facades/client/applicationoffers/base.go

```diff
@@ -51,7 +53,7 @@ func (api *BaseAPI) checkControllerAdmin() error {
 }
 
 // modelForName looks up the model details for the named model and returns
-// the model (if found), the absolute model model path which was used in the lookup,
+// the model (if found), the absolute model path which was used in the lookup,
 // and a bool indicating if the model was found,
 func (api *BaseAPI) modelForName(modelName, ownerName string) (Model, string, bool, error) {
```

Comment:
Let's do this in a future PR and think about the right way to do it. It may make sense to add more functions to the model service (e.g. a ModelForName func). This PR already has a lot of churn.